src/backend/db/products.js

```diff
@@ -31,6 +31,7 @@ export const products = [
     title: "Adidas Men's T-Shirts",
     img: productimg1,
     price: 499,
+    Quantity: 1,
```

Comment:
This quantity part can be handled by the Mockbee backend also. Refer to docs and API controllers for more info.